faq/cockpit.mdx

```diff
@@ -27,7 +27,7 @@ Cockpit integrates seamlessly with Scaleway’s ecosystem. It provides pre-built
 
 ## How am I billed for using Cockpit with my Scaleway data?
 
-Scaleway’s monitoring is included for free for a period of 30 days  for metrics and 7 days for logs.
+Scaleway’s monitoring is included for free for a period of 31 days for metrics and 7 days for logs.
```

Comment:
```suggestion
Scaleway’s monitoring is included for free for a period of 31 days for metrics and 7 days for logs.
```
Is it always 31 days even if a month has less than 31 days?

Reply:
It is actually 30 days, I'll fix it locally.